### PR TITLE
Added solutions for the LC problems from 2880-2882

### LINKED ISSUE #534: [Feature Request]: Add leetcode solutions from 2880 to 2882
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature Description

Greetings Sir I would like to add the solutions of the Data science problems  of 2880, 28881 and 2882.

Thank you sir :)

### Use Case

This is for the Data Science Problems.

### Benefits

_No response_

### Add ScreenShots

_No response_

### Priority

Medium

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24 contributor
- [X] I have starred the repository

### PR DESCRIPTION
**Pull request by Abhay182005dat**
**Greetings !!**

I have succesfully added the solutions for the given LC problems sir . :)
Hence this closes #534 

## Description
The solutions are for these three Data Science problems :
- 2880 - Select Data
- 2881 - Create a new column
- 2882 - Drop Duplicate Rows

## Type of PR
- [👍] Feature enhancement
- [👍] Documentation update

### CODE REVIEW

File: dsa-solutions/lc-solutions/2500 - 3000/2880 - Select Data [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+id : Select Data
```
Comment (Ajay-Dhangar) [inline dsa-solutions/lc-solutions/2500 - 3000/2880 - Select Data:2]: replace `id : Select Data` to `id : 2880-select-data`

File: dsa-solutions/lc-solutions/2500 - 3000/2880 - Select Data [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
++---------+-----+
+| Ulysses | 13  |
++---------+-----+
+
+##Intuition
+I will say as this is a Data Science Problem the solution is also straightforward . Hope it helps :)
+.
+
+### Solution Code 
+<Tabs>
```
Comment (Ajay-Dhangar) [inline dsa-solutions/lc-solutions/2500 - 3000/2880 - Select Data:42]: complete your code because It is not full code so, now complite it

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]
now, update code based on suggestions and very important to complete every docs file because your code is not completed. So, first complete all.

**Comment by Abhay182005dat** [comment]
Done Sir :)

**Comment by Abhay182005dat** [comment]
Greetings Sir !!
 
I have completed updating about the Flex-Properties in thw Web-dev section

Hence this closes #572 

Thank You

File: web-dev/html/Flex-Box/Flex-Box Properties [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+        <div class="item" style="background-color: #FF5733;">Item 1</div>
+        <div class="item" style="background-color: #33FF99;">Item 2</div>
+        <div class="item" style="background-color: #3366FF;">Item 3</div>
+    </div>
+</body>
+</html>
+```
+## Conclusion
+Flexbox simplifies the process of creating layouts that adapt to different screen sizes and devices. By mastering Flexbox, you can create more responsive and dynamic web designs with less effort.
+
```
Comment (Ajay-Dhangar) [inline web-dev/html/Flex-Box/Flex-Box Properties:94]: delete

```md
Thank You 
-Created by Abhay :)
```

**Review by Ajay-Dhangar — COMMENTED** [review]
..

**Pull request merged**